docs/RFCS/20200401_alter_column_type.md

```diff
@@ -0,0 +1,87 @@
+- Feature Name: ALTER TABLE ... ALTER COLUMN ... SET DATA TYPE
+- Status: WIP
+- Authors: Richard Cai
+- Cockroach Issues: #9851
+
+# Table of Contents
+- [Summary](#summary)
+- [Motivation](#motivation)
+- [High-level summary](#high-level-summary)
+- [Detailed Explanation](#detailed-explanation)
+- [Problems](#Problems)
+- [Other considerations](#Other-considerations)
+- [Alternatives](#alternatives)
+
+# Summary
+Support changing type of a column. (For nontrivial type conversions using a cast).
```

Comment:
it should be noted that non trivial casts are ones that are not implicit casts
(i think all implicit casts in postgres by default can be alter tabled between one another based on my reading of https://www.postgresql.org/docs/current/sql-altertable.html)

Reply:
You're right - updated to use the correct terminology.